dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/versioning/VersionHistoryForm.java

```diff
@@ -136,6 +137,14 @@ private Division createMain(Body body) throws WingException
 
     private void createTable(Division main, VersionHistory history, boolean isItemView, Item item) throws WingException, SQLException
     {
+        Boolean isVisible = new DSpace().getConfigurationService().getPropertyAsType("versioning.item.history.include.submitter", Boolean.FALSE);
```

Comment:
This should likely use `getBooleanProperty` method instead of `getPropertyAsType`.

Also, rather than including `org.dspace.utils.DSpace`, you should just use `DSpaceServicesFactory.getInstance().getConfigurationService()` to get the ConfigurationService.